src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -287,7 +298,7 @@ void SetGridIC( real fluid[], const double x, const double y, const double z, co
    if ( Pres == NULL_REAL )
       Aux_Error( ERROR_INFO, "interpolation failed for pressure at radius %13.7e !!\n", r );
 
-   if ( CCSN_Prob == Post_Bounce )
+   if ( CCSN_Prob == Post_Bounce ||  CCSN_Prob == CoreCollapse )
```

Comment:
Add one more space before ||.

Also, entropy is not provided in the `s20_W2007`, the interpolated`Entr` is not well-defined.

Reply:
updated and I separated  `Post_Bounce` and `Core_Collapse` test.